sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/responses/AzureResponsesTestBase.java

```diff
@@ -230,4 +238,19 @@ public static void assertResponseItem(ResponsesItem responseItem) {
         assertNotNull(responseItem.getId());
         assertNotNull(responseItem.getType());
     }
+
+    public static void openImageFileBase64Runner(Consumer<String> testRunner) throws IOException {
+        byte[] imageBytes = Files.readAllBytes(Paths.get("src/test/resources/" + MS_LOGO_PNG));
+        String base64Image = Base64.getEncoder().encodeToString(imageBytes);
+        testRunner.accept(base64Image);
+    }
+
+    static void assertImageResponseForAzure(ResponsesResponse response) {
```

Comment:
We can make this method more specific to `assertAssistantMessage` and then do the casting in the test, instead of passing the full response here. 

I don't see though, any assertions related to Images here, so it might be good to rename this method.